src/lib.rs

```diff
@@ -162,6 +166,13 @@ pub struct RecoveryId(i32);
 #[derive(Copy, Clone, PartialEq, Eq)]
 pub struct Signature(ffi::Signature);
 
+/// A DER serialized Signature
+#[derive(Copy, Clone)]
+pub struct SerSignature {
+    data: [u8; 72],
+    len: usize,
+}
```

Comment:
`<bikeshedding>` 
`SerSignature` looks very weird to me. If you're not an expert, you may wonder what the difference between DER and SER is. ;) Suggestions: `SerializedSignature` or directly `DerSignature` (which kind of suggests that it's serialized)

Reply:
@apoelstra what do you think?
I think `SerSignature` is explanatory but maybe you're right and some people can find this confusing.

Reply:
Done